package-lock.json

```diff
@@ -4744,7 +4744,7 @@
     },
     "domelementtype": {
       "version": "1.3.1",
-      "resolved": "https://registry.npmjs.org/domelementtype/-/domelementtype-1.3.1.tgz",
+      "resolved": "http://registry.npmjs.org/domelementtype/-/domelementtype-1.3.1.tgz",
```

Comment:
:-1:

Reply:
Didnt change that manually. npm did that automatically. But yeah will undo this change

Reply:
No worries. Just undo this :)